convert2rhel/unit_tests/grub_test.py

```diff
@@ -403,17 +409,36 @@ def test_post_ponr_set_efi_configuration(
 
 
 @pytest.mark.parametrize(
-    ("is_efi", "subproc_called", "subproc", "exception"),
+    ("is_efi", "subproc_called", "subproc", "total_entries", "current_bootnum", "boot_order", "exception"),
```

Comment:
@SpyTec I've changed the parameters for the test a bit, this way we can pass different values for the parametrization